prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -0,0 +1,127 @@
+- name: "wa-phd"
+  description: "Washington Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "WA"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "wa-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml"
```

Comment:
remove metadata

Reply:
I would use "classpath:/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml" here unless you are actually uploading the transform to azure

Reply:
It will be uploaded to Azure in the validate ticket, that is how we are doing these.